app.py

```diff
@@ -89,7 +89,11 @@ def predict():
         proba=proba,
         observation=request.data,
     )
-    p.save()
+    try:
+        p.save()
+    except:
+        pass
+    
     return jsonify({'proba': proba})
```

Comment:
Also, if an error occurs, we should return a non-200 http code and an error describing what happened.